modules/primer-cards/README.md

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
I'm going to pull this into 10.4.0. Can you remove this file (primer-cards was deleted) and update for any new modules?